patternfly/patternfly-elements
# Maintain circled pfe-icon height in flex display

## Maintain circled pfe-icon height in flex display

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Aligned the pfe-icon-panel's flex-children (ex. pfe-icon) to prevent their height from stretching to parent's height.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Pull the branch down and type `run npm start` in your local environment
2. Navigate to http://localhost:8000/elements/pfe-icon-panel/demo/ and verify that the circled pfe-icon circles are not oblong :) 

#### Browser requirements
Your component should work in all of the following environments:


Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
